### DISCUSSION TITLE
Universal Groups

### DISCUSSION THREAD

**Discussion by OzGav** [music-assistant/support#1701] [Player Support]
Documentation https://music-assistant.io/player-support/universal/

**Comment by OTTIndia** [comment]
Limitations regarding large speaker groups ?

**Reply by OzGav** [reply]
As the docs say Universal Groups work on a best effort basis. You may or may not have a good experience with them depending on the mix of player types. Sync is not possible. So whilst this continues to be supported you are better off just grouping similar player types.

**Comment by OTTIndia** [comment]
How to own create docker run --network host --privileged -v

:/data ghcr.io/music-assistant/server

**Reply by OzGav** [reply]
Is this a question and what does it have to do with universal groups?

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
